src/wasm/cpp/tests/schemas.arcs

```diff
@@ -22,3 +22,11 @@ schema ServiceResponse
 schema SpecialFields
   Text for
   Number internal_id
+
+particle Dummy
```

Comment:
One last nit: since this name actually does appear (extensively) in the tests, maybe 'Dummy' wasn't a good choice on my part. I'd be tempted just to use 'X' for brevity, but I guess 'Test' would be better.

Reply:
I think that should be a quick find and replace. Not a problem.